lighthouse-core/test/audits/seo/tap-targets-test.js

```diff
@@ -152,43 +154,44 @@ describe('SEO: Tap targets audit', () => {
     assert.equal(failure.height, 10);
   });
 
-  it('fails if a tap target overlaps vertically', () => {
-    const auditResult = auditTapTargets(
+  it('fails if a tap target overlaps vertically', async () => {
+    const auditResult = await auditTapTargets(
       getBorderlineTapTargets({
         overlapBelow: true,
       })
     );
     assert.equal(auditResult.rawValue, false);
   });
 
-  it('fails when one of the client rects overlaps', () => {
-    const auditResult = auditTapTargets(
+  it('fails when one of the client rects overlaps', async () => {
+    const auditResult = await auditTapTargets(
       getBorderlineTapTargets({
         overlapSecondClientRect: true,
       })
     );
     assert.equal(auditResult.rawValue, false);
   });
 
-  it('reports 2 items if the main target is overlapped both vertically and horizontally', () => {
+  it('reports 2 items if the main target is overlapped both vertically and horizontally',
```

Comment:
super 🚲 🏠 but I'd avoid the line and indentation split by getting rid of "the main target is" since none of the other tests really need to specify that piece


😆